fe/fe-core/src/main/java/com/starrocks/load/loadv2/BrokerLoadJob.java

```diff
@@ -331,6 +334,67 @@ private void createLoadingTask(Database db, BrokerPendingTaskAttachment attachme
         }
     }
 
+    @Override
+    public void afterAborted(TransactionState txnState, boolean txnOperated, String txnStatusChangeReason)
```

Comment:
better move this function to `BulkLoadJob` because spark load job also need retry.

Reply:
I don't do the spark load job retry in this pr.